### ISSUE TITLE
Release Manager - Branch naming convention has changed in ODS 3.x in 

### ISSUE THREAD

**Issue by felipecruz91** [opendevstack/ods-core#849]
Hi,

I've noticed a slight difference in the branch naming convention when deploying a component to DEV using Release Manager between ODS 2.x and 3.x:

### ODS 2.x
`release/${version}`, i.e: `release/1.0`

![image](https://user-images.githubusercontent.com/15997951/96605210-34155480-12f6-11eb-94c8-e717d2df144a.png)

### ODS 3.x
`release/ods-generated-${version}`, i.e: `release/ods-generated-${version}`

![image](https://user-images.githubusercontent.com/15997951/96604695-ae91a480-12f5-11eb-9a4a-203d98fb0ddd.png)

However, I haven't found where in the official documentation this information is. These is the documentation I have checked:

- https://www.opendevstack.org/ods-documentation/opendevstack/3.x/quickstarters/release-manager.html
- https://www.opendevstack.org/ods-documentation/opendevstack/3.x/update-guides/3x.html

**Additional context**
This is the only issue I found with occurrences to "ods-generated": #484

/ cc @metmajer

**Comment by metmajer** [comment]
@michaelsauter I am not too happy with the "ods-generated" in every release branch, since it is in fact generated through a user. Is there a good reason to have it named this way?

**Comment by clemensutschig** [comment]
I changed it - to show people that this an ods owned one vs. user created
ones

Martin Etmajer <notifications@github.com> schrieb am Di., 20. Okt. 2020,
17:31:

**Comment by metmajer** [comment]
My point of view is users don't create releases manually and shouldn't. The Release Manager creates the releases, based on manual intervention. I see no reason in having `ods-generated`, especially if it's the general case. Safe to rename?

**Comment by clemensutschig** [comment]
But people create Branches by hand... ;-)

Martin Etmajer <notifications@github.com> schrieb am Mi., 21. Okt. 2020,
12:09:

**Comment by metmajer** [comment]
Yup, agree for feature and fix branches. Do you see users create release/* branches?

**Comment by clemensutschig** [comment]
Sure... We Cannon Block this.

Martin Etmajer <notifications@github.com> schrieb am Mi., 21. Okt. 2020,
14:18:

**Comment by metmajer** [comment]
We should have reflected that we are generating a `release` branch. `ods-generated` alone doesn't indicate the nature of such a branch. Suggestion: `release-ods-generated/${version}`. @clemensutschig I can hardly imagine that teams would use the Release Manager and manual releases in the same project, but this provides a safe ground at least. Hmm?

@borja44 is there any documentation update required for the ODS 3.1 release?

**Comment by clemensutschig** [comment]
Fine .

Martin Etmajer <notifications@github.com> schrieb am Mi., 21. Okt. 2020,
14:29:
